libs/account-lib/src/utils/blockchain-scanner.service.ts

```diff
@@ -89,7 +89,7 @@ export abstract class BlockchainScannerService {
         this.scanInProgress = false;
         return;
       }
-      this.logger.verbose(`Starting scan from block #${currentBlockNumber}`);
+      // this.logger.verbose(`Starting scan from block #${currentBlockNumber}`);
```

Comment:
These log messages are filling the log and don't generally seem to be useful.
Is it worthwhile to try and filter it somehow and log every 5-10 minutes or something like that?

Reply:
I'm fine leaving it out, but if so it should probably just be removed instead of commented.
Could also change it to `log.debug`--debug level should only be for Gateway developers, not Gateway users.
Either is good.

Reply:
At some point we should probably do some logging cleanup; specifically:
* DEBUG logs should only be relevant for developers working on Gateway itself
    * (understanding, though, that these first clients may need to turn on debug to help us resolve their issues)
* Need a way to turn on VERBOSE logs without also enabling DEBUG
* Consider moving to Pino (as other PL projects) or Winston (very popular) logger